configure.ac

```diff
@@ -610,5 +618,6 @@ Test suite
   complete test suite: ${have_tests_all}
     recommended tests with SquashFS ... ${have_tests_squash}
     documentation ... ${have_docs}
+    ShellCheck ≥ $vmin_shellcheck ... ${SHELLCHECK_VERSION_NOTE}
```

Comment:
Where is `$SHELLCHECK_VERSION_NOTE` being defined?

Reply:
Sorry, my comment got eaten. It is part of the magic of `CH_CHECK_VERSION`.